### ISSUE TITLE
font variable

### ISSUE THREAD

**Issue by muhajirdev** [hawezo/tailwindcss-theming#1]
Is it possibble to also change font variable?

**Comment by hawezo** [comment]
In the meantime, you can still do it manually. You'd have to add a `--font-title: Raleway, "Open Sans", Roboto;` variable to any theme class (`:root` for instance), and add the following Tailwind config:

```js
module.exports = {
  theme: {
    fontFamily: {
      title: ['var(--font-title)'],
    }
  },
};
```

**Comment by hawezo** [comment]
I found the time to work on the custom properties features. 

Now you can call `.variable('font-title', ['Roboto', '"Segoe UI"', 'Arial'])` on your theme to generate `:theme-name { --font-title: Roboto,"Seoge UI",Arial }` in it instead of manually having to write this variable in your CSS file. 

I still have to work on tying this feature to Tailwind's configuration.

**Comment by hawezo** [comment]
@muhajirdev Hey, I recently released v2.2.0 in which you can now properly have a `font` variable in your themes.

```js
new Theme()
  .variable('title', [ 'Roboto', '"Segoe UI"' ], 'fontFamily');
```

The third argument is the name of the Tailwind's core plugin you want to extend. In this case, a variable which name will be `--font-family-title` will be generated and the Tailwind utility `.font-title` will be created as well. :) 

It's a matter of time until I add a `.font()` and other aliases on the theme builder.
